uninstalling/ossm-uninstalling-openshift-service-mesh.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="ossm-uninstalling-openshift-service-mesh"]
+= Uninstalling OpenShift Service Mesh
+include::_attributes/common-attributes.adoc[]
+:context: ossm-uninstalling-openshift-service-mesh
+
+toc::[]
+
+You can uninstall {SMProductName} Operator from an existing {ocp-product-title} instance and remove its resources either by using the {ocp-product-title} web console, or through the command line interface (CLI).
```

Comment:
Normally I see Operators referred to as "_the_" `<name>` Operator, FWIW, but I guess either works:
```suggestion
You can uninstall the {SMProductName} Operator from an existing {ocp-product-title} instance and remove its resources either by using the {ocp-product-title} web console, or through the command line interface (CLI).
```